starters/adaptors/azure-swa/public/staticwebapp.config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "routes": [
+    {
+      "route": "/build/*"
+    },
+    {
+      "route": "/favicon.svg"
+    },
+    {
+      "route": "/manifest.json"
+    },
+    {
+      "route": "/.auth/*"
+    },
+    {
+      "route": "/*",
+      "rewrite": "/api/server"
+    }
```

Comment:
With that the route to "/" is not going to the function - it only serves the static index.html. 

You can see that here in my demo - the random number is always the same (the one created at build-time): https://salmon-dune-02a16c603-3.westeurope.2.azurestaticapps.net/ (here is the change for that https://github.com/derkoe/qwik-swa-demo/pull/3).

We have to define all routes that have to go to static files - otherwise Azure will serve the index.html.

Reply:
Adding the `/service-worker.js` to the routes fixes this issues. I have added that to the starter https://github.com/BuilderIO/qwik/pull/2384/commits/46dc606336ecee70609a7a0c01bea4dd3112df43

Here in the example: https://github.com/derkoe/qwik-swa-demo/pull/4

Reply:
Yes, I see, that makes sense. I hadn't taken into account the flower page, thanks for adding the `/service-worker.js` to the routes.